quantum/process_keycode/process_unicode_common.c

```diff
@@ -80,6 +83,10 @@ void persist_unicode_input_mode(void) { eeprom_update_byte(EECONFIG_UNICODEMODE,
 __attribute__((weak)) void unicode_input_start(void) {
     unicode_saved_caps_lock = host_keyboard_led_state().caps_lock;
 
```

Comment:
Unnecessary newline. The way I see it, the two assignments are logically grouped together.

```suggestion
```